build.gradle

```diff
@@ -48,8 +48,8 @@ dependencies {
     implementation(group: 'org.glassfish.jersey.media', name: 'jersey-media-jaxb', version: '2.39')
     implementation(group: 'com.google.guava', name: 'guava', version: '31.1.0.jre-redhat-00005')
     implementation(group: 'org.slf4j', name: 'slf4j-api', version: '1.7.36')
-    implementation(group: 'ch.qos.logback', name: 'logback-core', version: '1.4.14')
-    implementation(group: 'ch.qos.logback', name: 'logback-classic', version: '1.4.14')
+    implementation(group: 'ch.qos.logback', name: 'logback-core', version: '1.2.11')
```

Comment:
please add a comment on why we need to stick to this version, so future developers do not make the same mistake.

would also be good to comment on why we can't update springboot to latest patch (if it's not latest)

Reply:
Added a comment mentioning the issue.
springboot is at the latest patch version. https://mvnrepository.com/artifact/org.springframework.boot/spring-boot